app/src/main/java/app/grapheneos/camera/ui/activities/MainActivity.kt

```diff
@@ -726,6 +726,8 @@ open class MainActivity : AppCompatActivity(),
                     showMessage(
                         getString(R.string.please_wait_for_image_to_get_captured_before_sharing)
                     )
+                } else if (!isThumbnailLoaded) {
```

Comment:
This is not a proper fix, there's a lot more that needs to be done in thumbnail loading code. I'll handle it in my next PR.

Reply:
I can drop this.